packages/dd-trace/src/config.js

```diff
@@ -870,6 +866,25 @@ class Config {
     }
   }
 
+  _setSamplingRule (obj, name, value) {
```

Comment:
Why is this needed? It looks like cases where it should be a string VS an array are isolated.

Reply:
when it is null/undefined, we cannot do sample_rate -> sampleRate mapping on it